docs/patterns/compute-resource-consolidation.md

```diff
@@ -21,21 +21,21 @@ Consolidate multiple tasks or operations into a single computational unit. This
 
 ## Context and problem
```

Comment:
Can you update the metadata at the top of this file?  Updating the date if you've reviewed the content.

Reply:
Date updated